doc/classes/FBXDocumentExtensionConvertImporterMesh.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="FBXDocumentExtensionConvertImporterMesh" inherits="GLTFDocumentExtension" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
```

Comment:
Can add a description.

Reply:
how?
also is it that important?
this will mostly be an internal part of the engine and no-one will touch it manually right?

Reply:
Should still have a description, please add it as you are exposing the class

Reply:
> Should still have a description, please add it as you are exposing the class, if you are needing to expose the class, why is that needed btw?


u mean why exposing the class is needed? well no real reason other than that the GLTFDocumentExtensionConvertImporterMesh being exposed to the user for some reason im trying to make my code as "official" as i possibly can